### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by alimJalanTech**
Working on Twitter Clone:

- Sidebar
- TweetBox
- News Feed
- Widgets
- Firebase integration

 UI: 
image

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./App.css";
+import Sidebar from "./components/Sidebar";
+import Feed from "./components/Feed";
+import Post from "./components/Post";
+
+function App() {
```
Comment (roopaljasnani) [inline src/App.tsx:7]: Pls add TS return types

File: src/components/Feed.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Tweetbox from "./Tweetbox";
+import "./Feed.css";
+import Post from "./Post";
+
+const Feed = () => {
```
Comment (roopaljasnani) [inline src/components/Feed.tsx:6]: TS return type on all components pls

File: src/components/SidebarOptions.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./SidebarOptions.css";
+
+type SidebarOptionsProps = {
+  Icon: React.ElementType;
+  text: string;
+};
+
+const SidebarOptions = ({ Icon, text }: SidebarOptionsProps) => {
```
Comment (roopaljasnani) [inline src/components/SidebarOptions.tsx:9]: For React components, the recommended way of typing is 

```tsx
const SidebarOptions = ({ Icon, text }): React.FC<SidebarOptionsProps> => {
```

File: src/components/Feed.tsx (hunk 2 above)
Comment (alimJalanTech) [reply src/components/Feed.tsx:6]: Resolved

File: src/components/SidebarOptions.tsx (hunk 3 above)
Comment (alimJalanTech) [reply src/components/SidebarOptions.tsx:9]: Resolved

File: src/App.tsx (hunk 1 above)
Comment (alimJalanTech) [reply src/App.tsx:7]: Resolved

**Pull request merged**